### PR TITLE
fix: update graphql-request to fix unlock protocol

### PR DESCRIPTION
**Pull request by BillySh1**
## Description

Closes # MASK-697

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Review by Jack-Works — DISMISSED** [review]

**Review by hanszhoou — DISMISSED** [review]

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Comment by Jack-Works** [comment]
rebase & solves the conflict.

**Review by Jack-Works — DISMISSED** [review]

**Comment by wenluomask** [comment]
The unlock feature is still hidden state, I cannot test it.

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "@types/masknet__global-types": "workspace:*",
     "@types/react": "^17.0.39",
     "@types/react-dom": "^17.0.11",
+    "graphql-request": "^4.0.0",
```
Comment (Jack-Works) [inline package.json]: Why you need to install it as a top-level package?

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (BillySh1) [reply package.json]: yes, removed

**Review by Jack-Works — DISMISSED** [review]

**Review by guanbinrui — APPROVED** [review]

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Comment by wenluomask** [comment]
https://mask.atlassian.net/browse/MASK-697

**Pull request closed without merging**